Add Seth Shaw to the CLAW Committers list

ysuarez opened the pull request:
* Other Relevant Links: [CLAW call agenda/minutes](https://github.com/Islandora-CLAW/CLAW/wiki/May-16,-2018#agenda)

# What does this Pull Request do?

Updates Committers document to include our latest CLAW Committer.

# What's new?

Added info on new committer Seth Shaw.

# How should this be tested?

Build the docs using mkdocs following these instructions
http://islandora-claw.github.io/CLAW/technical-documentation/docs-build/

Then see the changes locally on 
http://localhost:8111/contributing/committers/

# Additional Notes:

This is only the commit that changes the documentation content. At this time I need the help of a CLAW committer to first [install mkdocs and build the CLAW docs](http://islandora-claw.github.io/CLAW/technical-documentation/docs-build/) AND THEN run the following mkdocs command...

`mkdocs gh-deploy --clean`

This command will then deploy the documentation content to the CLAW GitHub.io Pages inside the CLAW repo's gh-pages branch. Then the CLAW GitHub.io Pages will display the new content at this URL...

http://islandora-claw.github.io/CLAW/contributing/committers/

There might be a way for a CLAW non-committer to create a PR from CLAW repo fork, but I am still sorting out the workflow. Let me know if anyone has any suggestions.

# Interested parties
@Islandora-CLAW/committers

ysuarez commented:
@dannylamb  FYI, I had offered to add @Natkeeran to the list of CLAW committers last week, but it turns out that @manez had taken care of this on merged PR #753. At this point we just need a CLAW committer to run `mkdocs gh-deploy --clean` to deploy that documentation change (and the one I just committed) to the CLAW.io site.

Like I said in the above PR, there might be a way for a CLAW non-committer to create a PR from a CLAW repo fork, but I am still sorting out the workflow. Let me know if anyone has any suggestions.

whikloj commented:
@dannylamb Should we remove Nick from this list now? Or make a new Emeritus section?

ysuarez commented:
@whikloj & @dannylamb if you decide to make Nick an emeritus committer, I can write out the formatting for that. I can create new section/table like in the Islandora site, or we can also add something simple like an asterisk with an explanation below the current table.

dannylamb commented:
@ysuarez Yeah sure, while you're in there you may as well update @ruebot to committer emeritus status.  I'd just make a second table, there will eventually be others.

ysuarez commented:
@whikloj & @dannylamb I just added the emeritus section and moved Nick. 

I also ran the CLI spellchecker [aspell](https://en.wikipedia.org/wiki/GNU_Aspell) on committers.md. 

I will check some other CLAW doc pages.
